compiler/noirc_evaluator/src/brillig/brillig_gen/brillig_block.rs

```diff
@@ -55,6 +55,7 @@ impl<'block, Registers: RegisterAllocator> BrilligBlock<'block, Registers> {
         brillig_context: &'block mut BrilligContext<FieldElement, Registers>,
         block_id: BasicBlockId,
         dfg: &DataFlowGraph,
+        call_stacks: &mut CallStackHelper,
```

Comment:
I feel we could put this on the BrilligBlock itself but it is non-blocking

Reply:
The location tree is englobing all the blocks so it needs to be passed around the blocks, which is what we do here.
It seems more straightforward to me.

Reply:
Either way I will not block on this, so no need to update.